### ISSUE TITLE
Cannot use the layout buttons

### ISSUE THREAD

**Issue by szaimen** [nextcloud/integration_whiteboard#9]
Hi,

I've discovered that the layout buttons on the right side don't work. 

I get a Javascript error when clicking on one of them:
![image](https://user-images.githubusercontent.com/42591237/108887425-864ee500-760a-11eb-98e3-c88157c5e2e8.png)

Here is a screenshot of the buttons that I am talking about:
![image](https://user-images.githubusercontent.com/42591237/108887381-76370580-760a-11eb-85b4-88956a28e03e.png)

Please tell if you need more infos about my setup.

**Comment by szaimen** [comment]
Ah, I figured it out: it is a handling error from my side: the buttons only work if you select a few elements before.
But maybe there should be a toast saying that you need to select a few elements to be abe to use the layout buttons instead of just bugging out?

**Comment by socialize-IT** [comment]
Yes you got it right. This is a nice idea, but a limitation of the used spacedeck system..

**Comment by sudwhiwdh** [comment]
If it's important to you, see if there's already an issue about it, or fill out a new one at https://github.com/spacedeck/spacedeck-open/issues ;)

**Comment by sudwhiwdh** [comment]
If it's important to you, see if there's already an open issue about it, or fill out a new one at https://github.com/spacedeck/spacedeck-open/issues ;)
